helix-core/pom.xml

```diff
@@ -159,6 +159,10 @@ under the License.
       <artifactId>metrics-core</artifactId>
       <version>3.2.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.helix</groupId>
+      <artifactId>helix-zk</artifactId>
```

Comment:
Do we really want to call it "helix-zk"? I thought the idea was to have a module fully decoupled from Helix's logic. In that case, names like "zookeeper-core" or "zk-api" might be more appropriate?

Reply:
How about just call it zookeeper-client?

Reply:
In my opinion, since this module is still in helix like a helper and not a separate project, I would go with helix-zookeeper.

Reply:
my suggestion is also to not include any helix* keywords in this module, maybe just zookeeper, or zookeeper-client, or zookeeper-api, etc.